### PR TITLE
Instructions for adding a language

### PR DESCRIPTION
**Pull request by whyboris**
And integrating French into the app 👍  Thanks @cal2195 for the script and for the French translation!

### CODE REVIEW

File: src/app/i18n/README.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+3. The instructions in the script will inform you:
+> copy to_translate.txt contents into Google Translate, and paste the translations into a file called translated.txt
+4. Once done, press the `Enter` key in your terminal and you're done!
+5. Open a PR (Pull Request) with the changes.
+
+## Integrating the language into the app
+
+You're welcome to open a Pull Request as is and I'll implement the language into the app. Alternatively, you can edit just a few more files and you'll be ready! It's rather eays -- just follow the patterns!
+
+1. The newly generated file will need a change: `export const English = {` -> to the appropriate language name
```
Comment (cal2195) [inline src/app/i18n/README.md]: This is supposed to happen with the script - does it not?

Comment (whyboris) [reply src/app/i18n/README.md]: It _should_ happen in the script 😅 -- I think it doesn't -- checking again now

Comment (whyboris) [reply src/app/i18n/README.md]: ps - I also removed all the `tslint` rules from within the file -- and I'm keeping only one rule at the top of the file. Might make the script simpler (if you're editing it).

Comment (cal2195) [reply src/app/i18n/README.md]: It's the scripts fault! I'll fix! 👍

Comment (whyboris) [reply src/app/i18n/README.md]: Seems to not put the language name into the `ts` file still - but it's likely not worth the effort to fix 👌

**Pull request merged**